### PR TITLE
itermocil 0.1.0

### PR DESCRIPTION
**Pull request by TomAnthony**
iTermocil allows you setup pre-configured layouts of windows and panes in iTerm2, having each open in a specified directory and execute specified commands. It is compatible with termocil files designed for tmux sessions.

I understand it is generally not perfect for authors to submit their own work. However, none of the 'Interesting Taps & Branches' seemed appropriate, and this is a relatively clean little utility.  Let me know if I should be looking elsewhere.

### CODE REVIEW

File: Library/Formula/itermocil.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+class Itermocil < Formula
+  homepage "https://github.com/TomAnthony/itermocil"
+  url "https://github.com/TomAnthony/itermocil/archive/0.1.0.tar.gz"
+  version "0.1.0"
```
Comment (bfontaine) [inline Library/Formula/itermocil.rb]: You can remove this line.

File: Library/Formula/itermocil.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  sha256 "464902145739f10155b3ba2c5482ca5e5f76032455d2935528d67421a03c7931"
+
+  resource "PyYAML" do
+    url "https://pypi.python.org/packages/source/P/PyYAML/PyYAML-3.11.tar.gz"
+    sha1 "1a2d5df8b31124573efb9598ec6d54767f3c4cd4"
```
Comment (bfontaine) [inline Library/Formula/itermocil.rb]: Please use a `sha256` checksum.

File: Library/Formula/itermocil.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
+class Itermocil < Formula
+  homepage "https://github.com/TomAnthony/itermocil"
```
Comment (bfontaine) [inline Library/Formula/itermocil.rb]: Please add a short description using `desc "…"` above this line.

File: Library/Formula/itermocil.rb [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
+    %w[PyYAML].each do |r|
+      resource(r).stage do
+        system "python", *Language::Python.setup_install_args(libexec/"vendor")
+      end
+    end
+
+    ENV.prepend_create_path "PYTHONPATH", libexec
+
+    libexec.install Dir["PyYAML"]
```
Comment (bfontaine) [inline Library/Formula/itermocil.rb]: Use `libexec.install "PyYAML"`

**Comment by bfontaine** [comment]
It’s a self-submission of new and not-yet popular project, so we’re probably going to pass on this. However, you can host it in your own [tap](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/brew-tap.md).

The fastest way is to create a `homebrew-something` repo, and put your formula under a `Formula` directory. Push it to GitHub, and *voilà*, people can install it with `brew install TomAnthony/something/itermocil` (replace `something` with whatever name you want).

**Comment by TomAnthony** [comment]
@bfontaine Thanks for the quick and useful feedback. I was just working through the Jenkins feedback but yours is better. :)

I'll create a tap (for now -- I'll be back ;). Thanks!

**Comment by DomT4** [comment]
Yeah, going to pass here.

```
 * GitHub repository not notable enough (<10 forks, <10 watchers and <20 stars)
 * GitHub repository too new (<30 days old)
```

New formula submissions should be audited in advance with `brew audit /path/to/formula --online --strict`. Thanks for the submission, We'll happily look at this again after the minimum requirements are fulfilled.

**Pull request closed without merging**